client-tools/Import/ImportHelper.cpp

```diff
@@ -982,7 +987,7 @@ void ImportHelper::addLastField(char const* field, size_t fieldLength,
                       << "In --merge-attributes: No matching value for "
                          "attribute name "
                       << attrProperties.value << " to populate attribute "
-                      << key;
+                      << *key;
```

Comment:
I think a better way is to capture `key` by reference and print it here as it was before, instead of taking its address and dereferencing it.
```suggestion
                      << key;
```